.travis.yml

```diff
@@ -2,6 +2,8 @@ script: "bundle exec rake neo4j:install[community-2.3.0] neo4j:start default --t
 language: ruby
 cache: bundler
 sudo: false
+jdk:
+  - oraclejdk8
```

Comment:
I think it is a good idea to run test suite on several Neo4j versions and it can be done in another PR. I created #17 for this.

For now I think that this can be removed because 2.3.0 version is used